repo_health/index/static/client/src/components/search/search.spec.js

```diff
@@ -10,27 +10,69 @@
 * This is the test file for the search component
 */
 
+const module = angular.mock.module;
+
 describe('Search', () => {
-  beforeEach(angular.mock.module('components.search'));
+  let sandbox;
+
+  beforeEach(module(
+    'app.resources', 
+    'components.repo-details',
+    'components.search')
+  );
 
   describe('SearchController', () => {
     let $componentController;
+    let $rootScope;
+    let RepoDetailsService;
+    let $q;
     let controller;
 
     beforeEach(inject(($injector) => {
       $componentController = $injector.get('$componentController');
+      $rootScope = $injector.get('$rootScope');
+      $q = $injector.get('$q');
+      RepoDetailsService = $injector.get('RepoDetailsService');
       controller = $componentController('search');
     }));
 
     describe('constructor', () => {
       it('should setup the controller', () => {
-        expect(controller).toBeDefined;
+        expect(controller).toBeDefined();
+      });
+      it('should setup loadingRepo and error', () => {
+        expect(controller.loadingRepo).toBeFalsy();
+        expect(controller.error).toBeNull();
       });
     });
 
     describe('getStats', () => {
-      it('should return the stats', () => {
-        //Add tests when function is implemented
+      let getStats;
+
+      it('should not make a call if url is invalid', () => {
+        spyOn(controller.RepoDetailsService, 'getStats');
+        controller.githubUrl = 'This not a url';
+        controller.getStats();
+        expect(controller.loadingRepo).toBeTrue;
```

Comment:
Pulled in the changes and rebased my working branch and found this when I looking through the files.  Should be `toBeTruthy()`. We can get it in the next PR.